api/courses.go

```diff
@@ -437,20 +435,20 @@ func createLecture(c *gin.Context) {
 			log.WithError(err).Error("Can't create folder for premiere")
 			return
 		}
-		// Copy file to shared storage
-		file, err := os.Create(fmt.Sprintf("%s/%s", premiereFolder, premiereFileName))
-		if err != nil {
-			log.WithError(err).Error("Can't create file for premiere")
-			c.AbortWithStatus(http.StatusInternalServerError)
-			return
-		}
-		reqFile, _ := req.File.Open()
-		_, err = io.Copy(file, reqFile)
-		if err != nil {
-			log.WithError(err).Error("Can't write file for premiere")
-			c.AbortWithStatus(http.StatusInternalServerError)
-		}
-		_ = file.Close()
+		//// Copy file to shared storage
```

Comment:
I would argue if this isn't needed right now it can be deleted, or refactored into a separate function.

Reply:
@joschahenningsen do you need that comment to implement the new upload function? Otherwise I would remove it 👍

Reply:
will follow up on this